cockatrice/src/player.cpp

```diff
@@ -699,24 +727,36 @@ void Player::retranslateUi()
         aViewHand->setText(tr("&View hand"));
         aViewTopCards->setText(tr("View &top cards of library..."));
         mRevealLibrary->setTitle(tr("Reveal &library to..."));
-        mRevealTopCard->setTitle(tr("Reveal t&op cards to..."));
+        mRevealTopCard->setTitle(tr("Reveal &top cards to..."));
+        topLibraryMenu->setTitle(tr("&Top of library..."));
+        bottomLibraryMenu->setTitle(tr("&Bottom of library..."));
         aAlwaysRevealTopCard->setText(tr("&Always reveal top card"));
-        aAlwaysLookAtTopCard->setText(tr("Al&ways look at top card"));
-        aOpenDeckInDeckEditor->setText(tr("O&pen deck in deck editor"));
+        aAlwaysLookAtTopCard->setText(tr("&Always look at top card"));
+        aOpenDeckInDeckEditor->setText(tr("&Open deck in deck editor"));
         aViewSideboard->setText(tr("&View sideboard"));
         aDrawCard->setText(tr("&Draw card"));
         aDrawCards->setText(tr("D&raw cards..."));
         aUndoDraw->setText(tr("&Undo last draw"));
         aMulligan->setText(tr("Take &mulligan"));
         aShuffle->setText(tr("&Shuffle"));
-        aMoveTopToPlay->setText(tr("Play top card"));
+
+        aMoveTopToPlay->setText(tr("&Play top card"));
         aMoveTopToPlayFaceDown->setText(tr("Play top card &face down"));
         aMoveTopCardToGraveyard->setText(tr("Move top card to grave&yard"));
         aMoveTopCardToExile->setText(tr("Move top card to e&xile"));
         aMoveTopCardsToGraveyard->setText(tr("Move top cards to &graveyard..."));
         aMoveTopCardsToExile->setText(tr("Move top cards to &exile..."));
         aMoveTopCardToBottom->setText(tr("Put top card on &bottom"));
-        aMoveBottomCardToGrave->setText(tr("Put bottom card &in graveyard"));
+
+        aDrawBottomCard->setText(tr("&Draw bottom card"));
+        aDrawBottomCards->setText(tr("D&raw bottom cards..."));
+        aMoveBottomToPlay->setText(tr("&Play bottom card"));
+        aMoveBottomToPlayFaceDown->setText(tr("Play bottom card &face down"));
+        aMoveBottomCardToGraveyard->setText(tr("Put bottom card &in graveyard"));
```

Comment:
```suggestion
        aMoveBottomCardToGraveyard->setText(tr("Move bottom card &in graveyard"));
```

Reply:
just tested this, the alt + letter shortcuts that work on the top menus for some reason don't allow me to select further options with alt + letter shortcuts, this is rather annoying.

Reply:
pressing alt just closes the just opened menu, I don't know why, this used to work and suddenly it doesn't, this could be a regression in qt or just a change in my window manager.

Reply:
oh, it does work and I'm just confused because darkmingo's theme doesn't properly show if options are available or not, greyed out options have the same color, guess I can open a ticket on that.
also note that you got to hold alt while navigating.